ci/scripts/matlab_build.sh

```diff
@@ -25,6 +25,6 @@ source_dir=${base_dir}/matlab
 build_dir=${base_dir}/matlab/build
 install_dir=${base_dir}/matlab/install
 
-cmake -S ${source_dir} -B ${build_dir} -G Ninja -D MATLAB_BUILD_TESTS=ON -D CMAKE_INSTALL_PREFIX=${install_dir} -D MATLAB_ADD_INSTALL_DIR_TO_SEARCH_PATH=OFF
+cmake -S ${source_dir} -B ${build_dir} -G Ninja -D MATLAB_ARROW_INTERFACE=ON -D MATLAB_BUILD_TESTS=ON -D CMAKE_INSTALL_PREFIX=${install_dir} -D MATLAB_ADD_INSTALL_DIR_TO_SEARCH_PATH=OFF
```

Comment:
How about adding newlines for readability?

```suggestion
cmake \
  -S ${source_dir} \
  -B ${build_dir} \
  -G Ninja \
  -D MATLAB_ARROW_INTERFACE=ON \
  -D MATLAB_BUILD_TESTS=ON \
  -D CMAKE_INSTALL_PREFIX=${install_dir} \
  -D MATLAB_ADD_INSTALL_DIR_TO_SEARCH_PATH=OFF
```

Reply:
This seems like a nice change. Thanks!